content/en/agent/proxy.md

```diff
@@ -1171,9 +1205,15 @@ network_devices:
             logs_dd_url: nginx.example.com:3845
             # Comment the line below to use encryption between the Agent and NGINX
             logs_no_ssl: true
-
+            
+#remove this section if your environment does not support Remote Configuration
+remote_config:
+    logs_dd_url: nginx.example.com:3846
+    # Comment the line below to use encryption between the Agent and NGINX
+    logs_no_ssl: true
```

Comment:
@bawsome can you confirm that's how remote config in the agent can be configured to use custom endpoints?

In any case, similar changes are needed under `HAProxy` -> `Agent`

Reply:
@olivielpeau good catch. I had missed this and we will add an example under HAProxy > Agent as well.
Also on your first Q, Arthur has provided the correct RC use for NGINX, so we will use that.

Reply:
This has been updated now.